src/skills-builder/skills-builder-modal/SkillsBuilderModal.jsx

```diff
@@ -50,13 +76,15 @@ const SkillsBuilderModal = () => {
 
         <ModalDialog.Body>
           <Container size="md" className="p-4.5">
-            <Stepper.Step eventKey={STEP1} title={formatMessage(messages.selectPreferences)}>
-              <SelectPreferences />
-            </Stepper.Step>
+            <Form>
+              <Stepper.Step eventKey={STEP1} title={formatMessage(messages.selectPreferences)}>
+                <SelectPreferences />
+              </Stepper.Step>
 
-            <Stepper.Step eventKey={STEP2} title={formatMessage(messages.reviewResults)}>
-              <ViewResults />
-            </Stepper.Step>
+              <Stepper.Step eventKey={STEP2} title={formatMessage(messages.reviewResults)}>
+                <ViewResults />
+              </Stepper.Step>
+            </Form>
```

Comment:
Wrapping the Skills Builder in a `Form` component — a minor update that was missed initially, does not effect UI but should have a positive impact on Accessibility

Reply:
yes that was my thinking as well